code/game/machinery/computer/crew.dm

```diff
@@ -239,7 +239,10 @@ GLOBAL_DATUM_INIT(crewmonitor, /datum/crewmonitor, new)
 		if (id_card)
 			entry["name"] = id_card.registered_name
 			entry["assignment"] = id_card.assignment
-			entry["ijob"] = jobs[id_card.real_title] //SKYRAT EDIT - ALTERNATE JOB TITLES, original = entry["ijob"] = jobs[id_card.assignment]
+			if(id_card.real_title)//SKYRAT EDIT - ALTERNATE JOB TITLES
```

Comment:
Just use `jobs[id_card.real_title || assignment]`

Reply:
huh

yeah I didn't know that's a thing but cool